code/mobile/mobile-common/Utils/RenderUtils.cpp

```diff
@@ -105,7 +105,7 @@ void Renderer::initProg(vector<pair<GLuint, bool>> texs, std::string vs, string
 
 // This is the actuall Constructor
 Renderer::Renderer(vector<pair<GLuint, bool>> texs, string frag, float xStart, float xEnd)
-        : rtt(false), frmBuf(0), maxLod(0){
```

Comment:
Why this change ?

Reply:
Change was done In order to fix compiler warnings but committed unknowingly.